torch/_refs/__init__.py

```diff
@@ -654,6 +697,22 @@ def float_power(
     return prims.pow(a, b)
 
 
+def _trunc_divide(
```

Comment:
Let's model this as an actual ref and have it point to a renamed "div_trunc" OpInfo that just renames the current "trunc_rounding" variant OpInfo for div to "div_trunc"

Reply:
let me know if you have questions about this because it can be a little tricky to manipulate OpInfos like this